enhancements/scheduling/descheduler-profiles.md

```diff
@@ -0,0 +1,259 @@
+---
+title: scheduling-profiles
+authors:
+  - "@damemi"
+reviewers:
+  - "@soltysh"
+  - "@ingvagabund"
+approvers:
+  - "@soltysh"
+  - "@ingvagabund"
+creation-date: 2020-11-23
+last-updated: 2020-11-23
+status: provisional
+see-also:
+  - "https://github.com/openshift/enhancements/pull/541/"
```

Comment:
I don't expect the other PR not merging, so I'd just link to the file like below.